terraform/groups/ecs-service/profiles/staging-eu-west-2/staging/vars

```diff
@@ -0,0 +1,11 @@
+environment = "staging"
+aws_profile = "staging-eu-west-2"
+
+# service configs
+use_set_environment_files = false
```

Comment:
I think this flag has to be set to true. Isn't it? Otherwise the .env file won't get passed to the task environment file.